lib/engine/tile.rb

```diff
@@ -214,6 +214,83 @@ def inspect
       "<#{self.class.name}: #{name}, hex: #{@hex&.name}>"
     end
 
+    # returns array where the value at an index is which edge that city (based
+    # on index in self.cities) should be rendered
+    def edges_for_city_rendering
+      @_edges_for_city_rendering ||= {}
+      cached_val = @_edges_for_city_rendering[@rotation]
+      return cached_val unless cached_val.nil?
```

Comment:
i prefer the @edges_for_city_rendering ||= begin 
end cache

if you want to keep this, invert it and do return cached_val if cached_val

Reply:
Resolved in 6bcca6b and 41238cf